examples/04_New Models.ipynb

```diff
@@ -4,14 +4,38 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# 4. Defining new Physics-Based Prognostic Models"
+    "# 4. Defining New Physics-Based Prognostic Models"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "All of the past sections describe how to use an existing model. In this section we will describe how to create a new model. This section specifically describes creating a new physics-based model. For training and creating data-driven models see 5. Data-driven Models."
+    "All of the previous sections describe how to use an existing model. In this section, we will explore how to create a new physics-based model. \n",
+    "\n",
+    "A physics-based model is a model where behavior is described by the physics of the system. Physics-based models are typically parameterized, so that exact behavior of the system can be configured or learned (through parameter estimation). \n",
+    "\n",
+    "For training and creating data-driven models, see __[05 Data Driven](05_Data%20Driven.ipynb)__."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Table of Contents\n",
+    "* [Linear Models](#Linear-Models)\n",
+    "* [New State Transition Models](#New-State-Transition-Models)\n",
+    "* [Derived Parameters](#Derived-Parameters)\n",
```

Comment:
Organization thought. Maybe we should put all these features into a section for ease of navigation. So the sections would be:

* Linear Models
* State Transition Models
* Advanced Features - with everything there as subsections
* Simplified Battery Model Example
* Conclusion

What do you think?

Reply:
Having an advanced features section makes sense since the subsections like derived parameters, matrix data access, state limits, custom events, and serialization all focus on optional features for the models. 

I'm assuming direct models should be its own section instead of a subsection?